src/Monolog/Handler/SendGridHandler.php

```diff
@@ -20,81 +21,72 @@
  */
 class SendGridHandler extends MailHandler
 {
+    private const CONTENT_TYPE = 'Content-Type: application/json';
+
     /**
      * The SendGrid API User
      */
     protected string $apiUser;
-
-    /**
-     * The SendGrid API Key
-     */
-    protected string $apiKey;
-
-    /**
-     * The email addresses to which the message will be sent
-     */
-    protected string $from;
-
     /**
      * The email addresses to which the message will be sent
      * @var string[]
      */
     protected array $to;
 
     /**
-     * The subject of the email
-     */
-    protected string $subject;
-
-    /**
-     * @param string          $apiUser The SendGrid API User
-     * @param string          $apiKey  The SendGrid API Key
-     * @param string          $from    The sender of the email
-     * @param string|string[] $to      The recipients of the email
-     * @param string          $subject The subject of the mail
-     *
+     * @param list<string>|string $to
      * @throws MissingExtensionException If the curl extension is missing
      */
-    public function __construct(string $apiUser, string $apiKey, string $from, string|array $to, string $subject, int|string|Level $level = Level::Error, bool $bubble = true)
+    public function __construct(
+        string $apiUser,
+        protected readonly string $apiKey,
```

Comment:
turning a protected property into a protected readonly one is also a BC break. You need to keep it non-readonly.

Reply:
same for the subject